doc/api/http2.md

```diff
@@ -1701,16 +1727,755 @@ req.end('Jane');
+  path: '/status?name=ryan',
+  href: '/status?name=ryan' }
+```
+
+To extract the parameters from the query string, the
+`require('querystring').parse` function can be used, or
+`true` can be passed as the second argument to `require('url').parse`.
+Example:
+
+```txt
+$ node
+> require('url').parse('/status?name=ryan', true)
+Url {
+  protocol: null,
+  slashes: null,
+  auth: null,
+  host: null,
+  port: null,
+  hostname: null,
+  hash: null,
+  search: '?name=ryan',
+  query: { name: 'ryan' },
+  pathname: '/status',
+  path: '/status?name=ryan',
+  href: '/status?name=ryan' }
+```
+
+### Class: http2.Http2ServerResponse
+<!-- YAML
+added: REPLACEME
+-->
+
+This object is created internally by an HTTP server--not by the user. It is
+passed as the second parameter to the [`'request'`][] event.
+
+The response implements, but does not inherit from, the [Writable Stream][]
+interface. This is an [`EventEmitter`][] with the following events:
+
+### Event: 'close'
+<!-- YAML
+added: REPLACEME
+-->
+
+Indicates that the underlying [`Http2Stream`]() was terminated before
+[`response.end()`][] was called or able to flush.
+
+### Event: 'finish'
+<!-- YAML
+added: REPLACEME
+-->
+
+Emitted when the response has been sent. More specifically, this event is
+emitted when the last segment of the response headers and body have been
+handed off to the HTTP/2 multiplexing for transmission over the network. It
+does not imply that the client has received anything yet.
+
+After this event, no more events will be emitted on the response object.
+
+### response.addTrailers(headers)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `headers` {Object}
+
+This method adds HTTP trailing headers (a header but at the end of the
+message) to the response.
+
+Attempting to set a header field name or value that contains invalid characters
+will result in a [`TypeError`][] being thrown.
+
+### response.connection
+<!-- YAML
+added: REPLACEME
+-->
+
+* {net.Socket}
+
+See [`response.socket`][].
+
+### response.end([data][, encoding][, callback])
+<!-- YAML
+added: REPLACEME
+-->
+
+* `data` {string|Buffer}
+* `encoding` {string}
+* `callback` {Function}
+
+This method signals to the server that all of the response headers and body
+have been sent; that server should consider this message complete.
+The method, `response.end()`, MUST be called on each response.
+
+If `data` is specified, it is equivalent to calling
+[`response.write(data, encoding)`][] followed by `response.end(callback)`.
+
+If `callback` is specified, it will be called when the response stream
+is finished.
+
+### response.finished
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+Boolean value that indicates whether the response has completed. Starts
+as `false`. After [`response.end()`][] executes, the value will be `true`.
+
+### response.getHeader(name)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `name` {string}
+* Returns: {string}
+
+Reads out a header that's already been queued but not sent to the client.
```

Comment:
small nit: long form `that has` (I’m only commenting because non-native speakers have a tendency to read `that's` as `that is`, not `that has`)